tools/delete_image_from_registry.bash

```diff
@@ -0,0 +1,27 @@
+# helper tool to delete tags from a docker registy
+# Need to run garbage collection afterwards in oder to free disk space https://docs.docker.com/registry/garbage-collection/
+# $> docker exec registry bin/registry garbage-collect --dry-run /etc/docker/registry/config.yml
+
+#https://stackoverflow.com/questions/25436742/how-to-delete-images-from-a-private-docker-registry
+if [$# -ne 4]; then
```

Comment:
This is invalid bash syntax and will never fire. The whitespace is important.
```suggestion
if [ $# -ne 4 ]; then
```

Reply:
Good catch :+1: